src/wasm.h

```diff
@@ -90,6 +90,9 @@ struct Name : public cashew::IString {
 
 // An index in a wasm module
 typedef uint32_t Index;
+inline Index toIndex(size_t index) {
```

Comment:
Method sounds good to me since there are going to be multiple uses. Can leave the others for a followup PR, I agree.

How about asserting on the right range in the function?